Connect Postgres DB in Circle CI with Django

albertcrowley opened the pull request:
* Adding a Postgres 14.0 DB container to the CI config
* Adding instructions for setting enviroment variables for DB conneciton in CircleCI
* Adding instructions for running CircleCI local CLI
* Modified the wait_for_db.py script to print output when successful.
* Change from using @retrying to @retry and set max retries for wait_for_db.py

albertcrowley committed:
Connect Postgres DB in Circle CI with Django
* Adding a Postgres 14.0 DB container to the CI config
* Adding instructions for setting enviroment variables for DB conneciton in CircleCI
* Adding instructions for running CircleCI local CLI
* Modified the wait_for_db.py script to print output when successful.
* Change from using @retrying to @retry and set max retries for wait_for_db.py

mjtravers commented:
@albertcrowley I think we should swap the Postgres 14 container out for Postgres 12 since that is the latest version supported by Cloud.gov.

See here on Cloud.gov supported verions: https://cloud.gov/docs/services/relational-database/

albertcrowley committed:
Setting Postgres version to 12 so we match cloud.gov version

albertcrowley committed:
Changing CircleCI postgres image to cimg/12.8 because cimg/12.0 does not exist

albertcrowley commented:
@mjtravers : I modified the CircleCI to use the newest Postgres 12 CircleCI image, which was "cimg/12.8".  I also updated the development docker image. I set that to "postgres:12" which should track with the most recent build of version 12.
